
The libpng*.so file should carry the -lz -lm dependency ...

  - Rene Rebe <rene@exactcode.de>

--- libpng-1.2.8/scripts/makefile.linux	2004-12-03 01:14:31.000000000 +0100
+++ libpng-1.2.8-fixed/scripts/makefile.linux	2005-04-02 16:29:40.000000000 +0200
@@ -99,7 +99,7 @@
 $(LIBNAME).so.$(PNGVER): $(OBJSDLL)
 	$(CC) -shared -Wl,-soname,$(LIBNAME).so.$(PNGMAJ) \
 	-o $(LIBNAME).so.$(PNGVER) \
-	$(OBJSDLL)
+	$(OBJSDLL) -lz -lm
 
 libpng.so.3.$(PNGMIN): $(OBJSDLL)
 	$(CC) -shared -Wl,-soname,libpng.so.3 \
